CHANGELOG.md

```diff
@@ -1,8 +1,22 @@
 # Changelog
 
 ## Current Develop Branch
+- [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
 
 ## 8.1.3 Mon Oct 26 2020
+- [#9642](https://github.com/MetaMask/metamask-extension/pull/9642) Prevent excessive overflow from swap dropdowns
+- [#9658](https://github.com/MetaMask/metamask-extension/pull/9658): Fix sorting Quote Source column of quote sort list
+- [#9667](https://github.com/MetaMask/metamask-extension/pull/9667): Fix adding contact with QR code
+- [#9674](https://github.com/MetaMask/metamask-extension/pull/9674): Fix ENS resolution of `.eth` URLs with query strings
+- [#9691](https://github.com/MetaMask/metamask-extension/pull/9691): Bump @metamask/inpage-provider from 6.1.0 to 6.3.0
+- Make the dropdown widgets for swaps keyboard accessible
```

Comment:
Should this have a PR associated with it?

Reply:
Whoops! That was an oversight. That's already covered by #9702 below. I'll remove this.

Reply:
Addressed in 4ca0f548e81faad16e45e998ce0f28d4f237d82a